pom.xml

```diff
@@ -169,6 +166,14 @@
     </contributor>
   </contributors>
```

Comment:
You could have your id here (and eventually some other contributors too), or, more more simplicity, we drop this section and only maintain the one in the website ?

Reply:
@mathieucarbou Given we use git, it makes sense to drop it entirely as that data can come from git.

Reply:
I'm leaving developer section since it has metadata to find us.  The contributor data as a whole did not other than names and as noted that can be found in git.

Reply:
actually one did have an email but still probably doesn't matter much.